src/countries.jrl

```diff
@@ -1,37 +1,247 @@
+p({"class":"foam.nanos.auth.Country", "code":"", "iso31661Code":"","name":""})
```

Comment:
What are these two for?

Reply:
Just a mistake from the script. Removed.
Also, do we need Kosovo? Some of the resources saying that it is not part of iso 3661-1
https://userapps.support.sap.com/sap/support/knowledge/en/2518366. However, some of the resources saying that the iso code is XXK

Reply:
leave for now and ask @kgrgreer